### PR TITLE
updates post feedback

### PR DESCRIPTION
**Pull request by isaac-florence**
#89

With updates/explanations on issue

### CODE REVIEW

**Review by nsbatra — COMMENTED** [review]
Hi Issac, thanks for updating this. Is there a reason why you added all those package names into the code? While it is sometimes necessary to have them, due to conflicts or to make *really* clear which package is being called, I think the code is generally easier to read without them.

File: new_pages/interactive_plots.Rmd [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Data visualisation is increasingly required to be interrogable by the audience. Consequently, is is becoming common to create interactive plots. There are several ways to include these but the two most common are **plotly** and **shiny**. 
 
-In this page we will focus on converting an existing `ggplot()` plot into an interactive plot with **plotly**. You can read more about **shiny** in the [Shiny and dashboards] page.  
+In this page we will focus on converting an existing `ggplot()` plot into an interactive plot with **plotly**. You can read more about **shiny** in the [Shiny and dashboards] page. What will be obvious but is worth mentioning nonetheless is that interactive plots are only useable in HTML format Rmarkdown   documents, not PDF or Word documents.
```
Comment (nsbatra) [inline new_pages/interactive_plots.Rmd:6]: May not be obvious at first to some of our more basic readers, so can we cut that word so they don't feel embarrassed? Can still communicate the point without that word obvious.

**Comment by isaac-florence** [comment]
I only added them because I thought that's what the feedback was asking for! Agree though it's not always necessary so can revert if you'd prefer, @nsbatra! I'd tend to agree that it reads more easily without them.

File: new_pages/interactive_plots.Rmd (hunk 1 above)
Comment (isaac-florence) [reply new_pages/interactive_plots.Rmd:6]: Good point. Will remove.

**Review by nsbatra — APPROVED** [review]

**Pull request merged**